naslib/utils/utils.py

```diff
@@ -196,6 +195,15 @@ def get_config_from_args(args=None, config_type="nas"):
         config.set_new_allowed(True)
         config.merge_from_list(args.opts)
 
+        if config.dataset == 'cifar10':
+            config.n_classes = 10
+        elif config.dataset == 'cifar100':
+            config.n_classes = 100
+        elif config.dataset == 'ImageNet16-120':
+            config.n_classes = 120
+        else:
+            config.n_classes = 10
```

Comment:
Could this conflict with e.g. custom datasets? 
We could raise an exception or create a warning instead
```suggestion
            raise AttributeError
```
```suggestion
            import warnings
            warnings.warn("Number of classes was not set. Default 10 is set.")
```

Reply:
Thanks for the suggestions added the warning.